README.md

```diff
@@ -1,2 +1,147 @@
-# firebase-tools-espanol
-Firebase Tools en Español  🖥️
+# Firebase (CLI) en Español
+
+#### Documentación de _firebase-tools_ en Español
+
+---
+
+Estas son las funcionalidades que ofrece la Línea de Comandos Firebase (CLI):
+
+* Desplegar código y archivos estáticos de tus proyectos Firebase
+* Ejecutar un servidor local para tu sitio en Firebase Hosting
+* Interactuar con los datos de tu base de datos Firebase
+* Importar/Exportar usuarios _a_ o _desde_ Firebase Auth
+
+Para empezar a utilizar Firebase (CLI), puedes leer a continuación la lista completa de comandos o chequear la [documentación específica para el Hosting](https://firebase.google.com/docs/hosting/quickstart).
+
+
+## Instalación
+
+Para instalar Firebase CLI, primero necesitas [estar registrado en Firebase](https://firebase.google.com/).
+
+Después necesitas instalar [Node.js](http://nodejs.org/) y [npm](https://npmjs.org/). Cabe destacar que instalar Node.js debe instalar npm también.
```

Comment:
Creo que está oración no se entiende muy bien

Reply:
Si, me parece que puede estar mejor.... ¿Cómo sería? 
La frase en inglés sería esta: 
`Then you need to install Node.js and npm. Note that installing Node.js should install npm as well. `

Yo creo que sería algo como: 
`Después necesitas instalar [Node.js](http://nodejs.org/) y [npm](https://npmjs.org/). Ten en cuenta que instalando Node.js tambien debería instalarse npm`

Reply:
Si así me parece mucho mejor @Villanuevand

Reply:
¡Hecho!  ahi lo modifico ...